.github/workflows/litex_sim.yml

```diff
@@ -76,24 +76,20 @@ jobs:
         uses: actions/checkout@v2
         with:
           repository: lschuermann/tock-litex
-          # Currently pointing to a fixed commit, switch to a tag on the next
-          # release with the Python requirements.txt included.
-          #
-          # Also, currently using the dev/litex-sim-ci branch, which
-          # pulls in a forked LiteX with GPIO support in the
-          # simulation and the simctrl modules
-          ref: da3f33968ae018c33dd10d705fd7e3ca04e5e056
+          # The pinned revision is different from the targeted release as
+          # documented in the LiteX boards, as the CI requires special patches
+          # to LiteX for interacting with the simulation:
+          ref: tock-cipin-1
```

Comment:
Assuming this ref is a name we control, might be nicer to name it based on a date? `tock-CI-pin-2022-08-18`? I assume these numbers will keep growing arbitrarily, so a little semantic context here might be nice.